index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+  <header id="header">
+    <img id="header-img" src="https://unsplash.com/photos/CglmcwSuMXQ" alt="" />
+    <nav id="nav-bar">
+      <a class="nav-link" href="#header-img">Nav1</a>
+      <a class="nav-link" href="#header-img">Nav2</a>
+      <a class="nav-link" href="#header-img">Nav2</a>
+    </nav>
+  </header>
+  <div class="container">
+    <div>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input type="email" id="email" name="email" /><br>
```

Comment:
I think it would be better to add a placeholder in the email input so that readers can know the format for writing an email.